tests/unit/modules/test_filter_info.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import pytest
+
+from ansible_collections.sensu.sensu_go.plugins.module_utils import (
+    errors, utils,
+)
+from ansible_collections.sensu.sensu_go.plugins.modules import filter_info
+
+from .common.utils import (
+    AnsibleExitJson, AnsibleFailJson, ModuleTestCase, set_module_args,
+)
+
+
+class TestilterInfo(ModuleTestCase):
```

Comment:
```suggestion
class TestFilterInfo(ModuleTestCase):
```

Found a nitpick 👏 🎆 🍾

Reply:
oops :)